### DISCUSSION TITLE
What's the difference between Fabric server and LAN world?

### DISCUSSION THREAD

**Discussion by SirEnder125** [FabricMC/fabric-api#2122] [Mod Dev Support] [answered]
Hey! I'm making a Minecraft mod using Fabric, and I just kept noticing the build configuration in the example mod called "Minecraft Server", so I googled it. Turns out that it works sorta like a LAN world if used correctly? I understand that it can also be extended to the internet like any Minecraft server. :thinking: But what are the differences between a Fabric Server and a LAN world other than that? Can a Fabric Server have plugins, for instance, Minecraft Skript, or is this not the case?

**Comment by frqnny** [comment, marked as answer]
It’s really just like people who run pure fabric servers. It’s only meant for testing stuff on a server, but there could be ways to open it to the internet.

**Reply by SirEnder125** [reply]
Oh, okay. Cool. :thinking: So it gives the developer the dedicated server experience, instead of... well... whatever there is in singleplayer/LAN?

**Reply by frqnny** [reply]
In a server environment, networking issues and desync issues will show up that don’t in the integrated singleplayer environment

**Reply by SirEnder125** [reply]
Okay, cool. Thanks!
